lib/queries.rb

```diff
@@ -0,0 +1,280 @@
+class Queries
+  def self.default
+    queries = {
+        "mostcommonlikers": {
```

Comment:
lets clean this up ... I like `user_participation` as a pattern... I think it is the right direction... maybe

`most-messages`, `top-likers`, `new-topics` ... looks kind of pretty in the url that way.

Reply:
Oh yes, I've cleaned this up now. It is so much better to look at when editing the sql :+1: